_includes/cockroachcloud/ccloud/backup-types.md

```diff
@@ -0,0 +1,6 @@
+[//]: # (Cloud backup types)
+
+Cockroach Cloud clusters support two types of backups:
+
+- Managed-service backups: automated full and incremental backups daily stored on Cockroach Cloud storage 
```

Comment:
I like the start to these! They're going to be very useful. I think there could be some adjustment to the text to make it clear that Cockroach Labs is running these backups and for what types of cluster. 

For managed-service backups:

> - Managed-service backups: Cockroach Labs takes automated backups of {{ site.data.products.serverless }} and {{ site.data.products.dedicated }} clusters that are stored in Cockroach Labs' cloud storage. 

I think we should also add either a link to the Restore Data from Backups page so customers can find the type and frequency of backup for both serverless and dedicated clusters. Something like: 

> "See the [Restore Data from Backups] page for the type and frequency of backups for {{ site.data.products.serverless }} clusters and {{ site.data.products.dedicated }} clusters."

OR

Add them here as sub-bullets. You'll find in my quick-fix PR #15537  some bullet points to these. It might be worth borrowing those for this include, or creating an include file for them as they will need to be updated regularly. Going with this option will mean waiting for my PR to merge and rebasing on master.

I'll leave you to see what makes sense on that front. I could see advantages to both ways. It might be good to see what these look like in a preview.